### ISSUE TITLE
newTempUser not being generated

### ISSUE THREAD

**Issue by cpuzzuol** [whitef0x0/node-email-verification#70]
#### What's this issue about?

When I run 

    rev.createTempUser()

the third argument "newTempUser" is null. The code I'm using is the exact same as in the tutorial.

#### Expected behavior

I was expecting a newTempUser to be created and passed to the createTempUser() callback.

#### Actual behavior

Instead, newTempUser is null.

#### Environment

Using express and mongoose

**Comment by ketanbhardwaj** [comment]
@cpuzzuol I am facing the same issue. Did you find any solution?

**Comment by emulibrary** [comment]
No, @ketanbhardwaj I have not had a chance to review and solve yet

**Comment by charisTheo** [comment]
same issue here though all three of the arguments of the callback function are null:
```
everif.createTempUser(user, function(err, existingPersistentUser, newTempUser){
        console.log("err: ", err);
        console.log("newTempUser: ", newTempUser);
        console.log("existingPersistentUser: ", existingPersistentUser);
} 
```
**err:  null
newTempUser:  null
existingPersistentUser:  null**

also using mongoose and express

**Comment by charisTheo** [comment]
if the user exists but has not confirmed yet, meaning that an email is sitting somewhere, the callback will return **null, null, null**

_from the index.js of email-verification at line 275, inside the createTempUser function:_

`// user has already signed up but not yet confirmed their account
        if (existingTempUser) {
          return callback(null, null, null);
        }`

make sure you handle this error correctly
